docs/kyma/docs/037-CRD.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Installer Custom Resource
+type: CRD
+---
+
+The installer Custom Resource (CR) file controls the Kyma installer, which is a proprietary solution based on the [Kubernetes operator](https://coreos.com/operators/). The file contains the basic information that defines Kyma installation. You can generate a custom CR file to install Kyma using specific parameters or a specific release in a `TAR.GZ` format.
+Find the custom resource template [here](../../../installation/resources/installer-cr.yaml.tpl).
```

Comment:
remember that for now links still do not work in the docs UI, especially link that point outside the docu folder, so please include the code snippet instead